include/z64.h

```diff
@@ -1221,7 +1046,7 @@ typedef struct GlobalContext {
     /* 0x01C24 */ ActorContext actorCtx;
     /* 0x01D64 */ CutsceneContext csCtx; // "demo_play"
     /* 0x01DB4 */ SoundSource soundSources[16];
-    /* 0x01F74 */ SubGlobalContext1F74 sub_1F74;
+    /* 0x01F74 */ Sram sram;
```

Comment:
this one is weird to me. I view srams "context" as the save context, and not as this struct with one u8 pointer in it, but I could just be looking at it from the wrong angle.
Is it even required to be a struct?

Reply:
It has to be a struct yeah, since it's the struct used by the sram system (you can see it as its `this` basically). It doesn't have much but I would still consider it as a usual context.